src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -9,7 +9,6 @@
     <!-- BUILDING_INBOX_LIBRARY is only false when building for netstandard to validate that the sources are netstandard compatible. -->
     <!-- This is meant to help with producing a source package and not to ship a netstandard compatible binary. -->
     <DefineConstants Condition="'$(TargetsNETStandard)' != 'true'">$(DefineConstants);BUILDING_INBOX_LIBRARY</DefineConstants>
-    <DefineConstants Condition="'$(TargetsNETStandard)' == 'true'">$(DefineConstants);USE_ABW_INTERNALLY</DefineConstants>
```

Comment:
Where were you using this Constant? I don't see that you actually removed it from code but I might have just missed it.

Reply:
See comment above: https://github.com/dotnet/corefx/pull/37017/files#r276954919

It's in `src/Common/src/System/Buffers/ArrayBufferWriter.cs`